webapp/portlet/src/main/webapp/vue-apps/user-setting-security/components/UserSettingSecurityWindow.vue

```diff
@@ -144,13 +144,20 @@ export default {
         return;
       }
 
+      if (this.newPassword === this.currentPassword) {
+        this.$refs.confirmNewPassword.setCustomValidity(this.$t('UserSettings.label.changePasswordIdentical'));
+        if (!this.$refs.form.$el.reportValidity()) {
+          return;
+        }
+      }
```

Comment:
As explained in the task, this verification must be done on server side. It will be use later to check more than one password in password history.
Please, can you redo this PR by making this check on server side